bin/babel-upgrade

```diff
@@ -1,9 +1,12 @@
 #!/usr/bin/env node
 
 const path = require('path');
-const { writePackageJSON, writeBabelRC } = require("../src");
+const { isAcceptedNodeVersion, writePackageJSON, writeBabelRC } = require("../src");
 const cwd = process.cwd();
-console.warn(`Current working directory: ${cwd}`);
+
+if (!isAcceptedNodeVersion()) {
+  console.warn("Please change Node version to 4 or higher");
```

Comment:
What do you think about just doing `throw new Error("Babel 7 will only support Node 4 and higher")`?

Reply:
👍